validator/stdeb.cfg

```diff
@@ -1,2 +1,17 @@
+# Copyright 2018 Intel Corporation
```

Comment:
Probably originally Intel copyright, but the file currently in master (if this gets rebased) was modified by Cargill.